config/database.yml

```diff
@@ -0,0 +1,28 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
+default: &default
+  adapter: postgresql
+  host: localhost
+  port: 5432
+  user: postgres
+  password: Maroc 1996
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+
+development:
+  <<: *default
+  database: blog_app
+
+# Warning: The database defined as "test" will be erased and
+# re-generated from your development database when you run "rake".
+# Do not set this db to the same as development or production.
+test:
+  <<: *default
+  database: db/test.postgresql
+
+production:
+  <<: *default
+  database: db/production.postgresql
```

Comment:
- [ ] Database names are not allowed to have dots. Kindly fix that.

  ![kglo](https://user-images.githubusercontent.com/81468675/196445420-fdcb01f6-84fe-4812-917d-e86e4343cf24.jpg)